hr_timesheet_report/i18n/fr.po

```diff
@@ -76,7 +76,6 @@ msgstr "Au moins un champ Détails doit être défini !"
 
 #. module: hr_timesheet_report
 #: code:addons/hr_timesheet_report/wizards/hr_timesheet_report_wizard.py:129
-#, fuzzy, python-format
```

Comment:
@sbidoul or here, either msgid is outdated or it's not even a format string 🤷‍♂️